File: CitizenDemo-DotNetCore/src/CitizenDemo.LoadGenerator/Models/Citizen.cs
using Newtonsoft.Json;

namespace CitizenDemo.LoadGenerator.Models
{
    public class Citizen
    {
        [JsonProperty(PropertyName = "citizenId")]
        public string? CitizenId { get; set; }

        [JsonProperty(PropertyName = "tenantId")]
        public string? TenantId { get; set; }

        [JsonProperty(PropertyName = "givenName")]
        public string? GivenName { get; set; }

        [JsonProperty(PropertyName = "surname")]
        public string? Surname { get; set; }

        [JsonProperty(PropertyName = "phoneNumber")]
        public string? PhoneNumber { get; set; }

        [JsonProperty(PropertyName = "streetAddress")]
        public string? StreetAddress { get; set; }

        [JsonProperty(PropertyName = "city")]
        public string? City { get; set; }

        [JsonProperty(PropertyName = "state")]
        public string? State { get; set; }

        [JsonProperty(PropertyName = "postalCode")]
        public string? PostalCode { get; set; }

        [JsonProperty(PropertyName = "country")]
        public string? Country { get; set; }

        public override string ToString()
        {
            return string.Format(
                "Citizen: CitizenId: {0}, GivenName: {1}, Surname: {2}, PhoneNumber: {3}, " +
                "StreetAddress: {4}, City: {5}, State: {6}, PostalCode: {7}, Country: {8}", 
                CitizenId, GivenName, Surname, PhoneNumber, StreetAddress, City, State, PostalCode, Country
                );
        }
    }
}

File: CitizenDemo-Java/src/loadgenerator/src/main/java/com/citizendemo/loadgenerator/models/Citizen.java
package com.citizendemo.loadgenerator.models;

public class Citizen {
    public String citizenId;
    public String givenName;
    public String surname;
    public String phoneNumber;
    public String streetAddress;
    public String city;
    public String state;
    public String postalCode;
    public String country;

    public Citizen() {
    }

    public Citizen(String citizenId, String givenName, String surname, String phoneNumber, String streetAddress, String city, String state, String postalCode, String country) {
        this.citizenId = citizenId;
        this.givenName = givenName;
        this.surname = surname;
        this.phoneNumber = phoneNumber;
        this.streetAddress = streetAddress;
        this.city = city;
        this.state = state;
        this.postalCode = postalCode;
        this.country = country;
    }

    public String toJSONString() {
        return "{" +
                "\"citizenId\":\"" + this.citizenId + "\"," +
                "\"givenName\":\"" + this.givenName + "\"," +
                "\"surname\":\"" + this.surname + "\"," +
                "\"phoneNumber\":\"" + this.phoneNumber + "\"," +
                "\"streetAddress\":\"" + this.streetAddress + "\"," +
                "\"city\":\"" + this.city + "\"," +
                "\"state\":\"" + this.state + "\"," +
                "\"postalCode\":\"" + this.postalCode + "\"," +
                "\"country\":\"" + this.country + "\"" +
                "}";
    }
}